requirements.txt

```diff
@@ -1,6 +1,6 @@
 # TODO: put back the standard pypi release when
 # next point version of openwisp-controller is released
-openwisp-controller @ https://github.com/openwisp/openwisp-controller/tarball/master
+openwisp-controller @ https://github.com/ManishShah120/openwisp-controller/tarball/issues/414-separate-serializer-classes
```

Comment:
I just merged https://github.com/openwisp/openwisp-controller/pull/416, now we can switch this to master